examples/neighbors/sparse-kde.py

```diff
@@ -221,8 +220,8 @@ def quick_shift_refinement(
         Index of the cluster centers in `X`
     labels : np.ndarray
         Labels of the input data, generated by `QuickShift`
-    normpks : float
-        Norm of the probability
+    probs : np.ndarray
```

Comment:
For correct linking in our rendered docs to the numpy docs use

```suggestion
    probs : numpy.ndarray
```

I think there might be a couple of other places in your code where this might apply.